recipes/bindash/meta.yaml

```diff
@@ -41,4 +41,7 @@ about:
 
 extra:
    identifiers:
-     - doi:10.1371/journal.pgen.1007758
+     - https://www.biorxiv.org/content/10.1101/2024.03.13.584875v1.abstract
+   maintainers:
+     - Jianshu Zhao
```

Comment:
please use github handles here

Reply:
I have added GitHub handle and ask for merge request again. Please merge if you find it ok.

Thanks,
Jianshu